dandi/bids_validator_xs.py

```diff
@@ -646,6 +646,34 @@ def select_schema_dir(
         )
 
 
+def log_errors(validation_result):
+    """
+    Raise errors for validation result.
+
+    Parameters
+    ----------
+    validation_result : dict
+        A dictionary as returned by `validate_all()` with keys including "schema_tracking",
+        "path_tracking", "path_listing", and, optionally "itemwise".
+        The "itemwise" value, if present, should be a list of dictionaries, with keys including
+        "path", "regex", and "match".
+    """
+    total_file_count = len(validation_result["path_listing"])
+    validated_files_count = total_file_count - len(validation_result["path_tracking"])
+    if validated_files_count == 0:
+        lgr.error("No valid BIDS files were found.")
+    if len(validation_result["schema_tracking"]) > 0:
```

Comment:
```suggestion
    if validation_result["schema_tracking"]:
```
would be more pythonic to the same effect, right?  but then why bother with this condition at all? just go into the `for` loop below and if it is empty -- nothing would happen.

the same below for `path_tracking`